src/Data/String/CodePoints.purs

```diff
@@ -143,19 +181,43 @@ countTail p s accum = case uncons s of
 -- | Drops the given number of code points from the beginning of the string. If
 -- | the string does not have that many code points, returns the empty string.
 -- | Operates in constant space and in time linear to the given number.
+-- |
+-- | ```purescript
+-- | >>> drop 5 "𝐀𝐀 b c"
+-- | "c"
+-- | -- compared to Data.String:
+-- | >>> drop 5 "𝐀𝐀 b c"
+-- | "b c" -- because "𝐀" is 2 chars long
```

Comment:
Perhaps it would be better to say "because "𝐀" occupies 2 code units" here? Just to avoid possible confusion over what "char" means.